### PR TITLE
kafka-connect-sorting-filtering

### PR DESCRIPTION
**Pull request by PLarboulette**
Hello, 

First PR on this repository to add a search bar in the Kafka Connect list. 

I use as an example the Schema Registry screen and implementation.

### CODE REVIEW

File: src/main/java/org/akhq/controllers/TopicController.java [hunk 1]
```diff
@@ -34,6 +34,7 @@
 import java.util.concurrent.ExecutionException;
 import java.util.stream.Collectors;
 import javax.inject.Inject;
+import org.akhq.models.Record;
```
Comment (PLarboulette) [inline src/main/java/org/akhq/controllers/TopicController.java:37]: I have compilation problems here, with ambiguous usage of Record (java.lang.Record and Record of package models). I import it explicitly.

File: src/test/java/org/akhq/controllers/TopicControllerTest.java [hunk 2]
```diff
@@ -9,7 +9,7 @@
 import org.akhq.utils.ResultPagedList;
 import org.apache.kafka.common.config.TopicConfig;
 import org.junit.jupiter.api.*;
-
+import org.akhq.models.Record;
```
Comment (PLarboulette) [inline src/test/java/org/akhq/controllers/TopicControllerTest.java:12]: I have compilation problems here, with ambiguous usage of Record (java.lang.Record and Record of package models). I import it explicitly.

**Comment by PLarboulette** [comment]
I have some problems with the "GroupControllerTest", function listApi() line 42 (expected: <1> but was: <5>) 
I can't understand why, I think I don't touch this part and this fail doesn't seem to be a problem for the global build.

**Comment by tchiotludo** [comment]
@PLarboulette : some test are flaky ;)
Last one works as you see !
Thanks for this one, will look at this soon ;)

**Review by tchiotludo — COMMENTED** [review]
Hi @PLarboulette 
Sorry for late response, really busy here ! 
I have made some test; the search is working fine :+1: 
But the pagination seems to be not working, the total page is not displayed, and changing to next page just keep the same result than previous one. 

Can you look at this ?

**Review by tchiotludo — CHANGES_REQUESTED** [review]
See comment above

**Comment by PLarboulette** [comment]
Hello, 

Oh sorry, It's pushed with a fix, I think it's better. 

FYI, during my work in Kafka Connect, I find a similar to mine : https://github.com/tchiotludo/akhq/pull/622

**Pull request merged**